### PR TITLE
Wind demo warning

### PR DESCRIPTION
**Pull request by 1chandu**
Windo demo is WebGL2 only application, display a message when run on non-webgl2 browsers.

Fixes : #975 

Testing: Runs fine on Chrome, Firefox, displays this message on Safari.

screen shot 2017-11-16 at 3 45 32 pm

### CODE REVIEW

**Comment by 1chandu** [comment]
This is on top of my other PR which isn't landed yet, will rebase this on master once it is landed, it should only include one last commit.

**Comment by 1chandu** [comment]
@jianhuang01 , yep, this will only include example change, as mentioned above, doc changes are from other approved PR which hasn't landed yet.

File: examples/wind/src/control-panel.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default class ControlPanel extends Component {
         { this._renderToggle('showParticles', 'particles') }
         { this._renderToggle('showWind', 'field') }
         { this._renderToggle('showElevation', 'elevation') }
-        { this._renderToggle('useDevicePixels', 'Retina/HD') }
+        { this._renderToggle('useDevicePixels', 'Retina/ HD') }
```
Comment (jianhuang01) [inline examples/wind/src/control-panel.js]: why do we need this space?

Comment (1chandu) [reply examples/wind/src/control-panel.js]: Just a non functional change, it looked better this way.

**Pull request merged**